step2-03/README.md

```diff
@@ -1,31 +1,48 @@
-# Step 2.3: Theming and Styling
+# Step 2.3: Theming and styling with UI Fabric
 
 [Lessons](../) | [Exercise](./exercise/) | [Demo](./demo/)
 
-Theming and Styling with UI Fabric. In this section, we will illustrate how to utilize some of the built-in theming and styling features right inside UI Fabric component library. 
+In this section, we will illustrate how to use some of the built-in theming and styling features of the UI Fabric component library.
 
-For advanced or non-Fabric component scenarios, UI Fabric also exposes its own CSS-in-JS library called `mergeStyles` that is very performant compared with other similar libraries. A CodePen that illustrates what `mergeStyles` does: https://codepen.io/dzearing/pen/jGdgrE?editors=1011
```

Comment:
This note being at the top seemed like it was left over from when the step didn't really cover mergeStyles. I moved the note contents and codepen link into the mergeStyles section.

Reply:
I made this distinction because people using Fabric components shouldn't generally be using `mergeStyles` directly.

Reply:
okay. I think elizabeth's wording is pretty clear for this point.